src/helpers/formatCreationDate.ts

```diff
@@ -0,0 +1,11 @@
+export const formatCreationDate = (date: string) => {
```

Comment:
This doesn't seem to work correctly

![image](https://github.com/elenapetra/react-app/assets/145835962/58510651-a064-46a8-8faa-835f7c953f68)

Reply:
Fixed